frontend/test/metabase/scenarios/dashboard-filters/reproductions/22482-round-relative-ranges.cy.spec.js

```diff
@@ -41,7 +41,7 @@ describe("issue 22482", () => {
     const expectedRange = getFormattedRange(
       moment()
         .startOf("month")
-        .add(-16, "month"),
+        .add(-14, "month"),
```

Comment:
Why did you have to change the test here? Doesn't seem related to the fix itself

Reply:
@kulyk Coding on a train with flaky internet and juggling branches. My bad, fixed.